ee/ucp/authorization/create-teams-with-ldap.md

```diff
@@ -36,23 +33,26 @@ before the members of the team are fully synced.
 ### Match Group Members (Direct Bind)
 
 This option specifies that team members should be synced directly with members
-of a group in your organization's LDAP directory. The team's membership will by
+of a group in your organization's LDAP directory. The team's membership will be
 synced to match the membership of the group.
 
-- **Group DN**: The distinguished name of the group from which to select users.
-- **Group Member Attribute**: The value of this group attribute corresponds to
+- **Group DN:** The distinguished name (DN) of the group from which to select users.
+- **Group Member Attribute:** The value of this group attribute corresponds to
   the distinguished names of the members of the group.
 
-
 ### Match Search Results (Search Bind)
 
 This option specifies that team members should be synced using a search query
 against your organization's LDAP directory. The team's membership will be
 synced to match the users in the search results.
 
-- **Search Base DN**: Distinguished name of the node in the directory tree where
+- **Search Base DN:** Distinguished name of the node in the directory tree where
   the search should start looking for users.
-- **Search Filter**: Filter to find users. If null, existing users in the search
+- **Search Filter:** Filter to find users. If null, existing users in the search
   scope are added as members of the team.
-- **Search subtree**: Defines search through the full LDAP tree, not just one
+- **Search subtree:** Defines search through the full LDAP tree, not just one
   level, starting at the Base DN.
```

Comment:
Would be great to have this information in a small table and not an overly-detailed bullet list.